### PR TITLE
Theme added

### PR DESCRIPTION
**Pull request by yurrpt**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -12,20 +12,21 @@ dependencies:
   flutter:
     sdk: flutter
 
-  cupertino_icons: 1.0.2
```
Comment (easeccy) [inline pubspec.yaml:15]: Revert.

Comment (yurrpt) [reply pubspec.yaml:15]: do i get line 15 back?

Comment (BRTZL) [reply pubspec.yaml:15]: yep also make sure you run `dart format .` and `flutter analyze` before pr

**Comment by easeccy** [comment]
Başka biri yapacak.

**Pull request closed without merging**